plugin.xml

```diff
@@ -10,22 +10,41 @@
     <keywords>cordova,urbanairship</keywords>
     <repo>https://github.com/urbanairship/urbanairship-accengage-cordova.git</repo>
     
+    <!-- ios -->
     <platform name="ios">
         <config-file parent="/*" target="config.xml">
             <feature name="urbanairship-accengage-cordova">
                 <param name="ios-package" value="urbanairship-accengage-cordova" />
             </feature>
         </config-file>
+        
+        <!-- Airship library -->
+        <podspec>
+            <config>
+                <source url="https://github.com/CocoaPods/Specs.git"/>
+            </config>
+            <pods use-frameworks="true">
+                <pod name="Airship/Accengage" spec="13.1.0" />
+            </pods>
+        </podspec>
+        
     </platform>
     
+    <!-- android -->
     <platform name="android">
         <config-file parent="/*" target="res/xml/config.xml">
-            <feature name="urbanairship-accengage-cordova">
-                <param name="android-package" value="urbanairship-accengage-cordova.urbanairship-accengage-cordova" />
+            <feature name="UAirshipPlugin">
+                <param name="android-package" value="com.urbanairship-accengage.cordova.UAirshipPlugin"/>
```

Comment:
```suggestion
                <param name="android-package" value="com.urbanairship.accengage-cordova.UAirshipPlugin"/>
```
It has to match whatever you put in the code, but it definitely should start with "com.urbanairship."